test/pg/postgres_test.exs

```diff
@@ -15,7 +15,7 @@ defmodule EctoEnum.PostgresTest do
 
   alias Ecto.Integration.TestRepo
 
-  test "accepts int, atom and string on save" do
+  test "accepts atom and string on save" do
     user = TestRepo.insert!(%User{status: :registered})
```

Comment:
Guess it needs to implement https://github.com/elixir-ecto/postgrex#extensions ?

Having similiar issues when trying to add test for #18 

Some references:
https://github.com/elixir-ecto/postgrex/issues/60
https://github.com/elixir-ecto/postgrex/issues/178
https://github.com/bryanjos/geo/blob/fa511e25963f8946ff7e1d7582ad645945773bbd/lib/geo/postgis/extension.ex